server/http/src/server.rs

```diff
@@ -204,8 +204,8 @@ mod tests {
 
                     // The output schema must include the input schema types
                     assert_eq!(
-                        ast::get_named_type(&input_schema.document, &"User".to_string()),
-                        ast::get_named_type(&output_schema.document, &"User".to_string())
+                        ast::get_named_type(&input_schema.document, &"User".to_string()).is_some(),
+                        ast::get_named_type(&output_schema.document, &"User".to_string()).is_some(),
```

Comment:
You could improve this test to be also be a test for `__typename` by asserting the full output schema.

Reply:
Yeah, I'll add a test, but `graphql/src/schema/api.rs` feels like the better place to add it — since that's where all the API schema tests live.

Reply:
I've added a better test for `__typename` being present in `graphql/src/schema/api.rs`.